options.html

```diff
@@ -7,16 +7,25 @@
     <style type="text/css">
       body {
           font: message-box;
-          text-rendering: optimizeLegibility;
           display: flex;
           flex-direction: column;
           max-width: 650px;
           margin: 0 auto;
       }
+      table {
+          margin-bottom: 3px;
+          width: 100%;
+      }
+      td:nth-child(1) {
+          width: 16em;
+      }
+      td:nth-child(2) {
+          width: 0;
+          padding-right: 2px;
+      }
       .button_container {
-          display: flex;
-          flex-direction: row;
           display: table-cell;
+          width: 58px;
```

Comment:
```suggestion
```
No fixed width, please. This one in particular causes the buttons to wrap on my system, I assume due to different fonts.

Reply:
Wrapping... that's not good!  What's your suggestion to provide sufficient room, but not have any extra blank space to the right of the buttons?

Reply:
I don't have one, unfortunately. I'd just remove the fixed width to avoid issues with different fonts or font sizes, and leave perfection for a bigger redesign.